scipy/stats/_morestats.py

```diff
@@ -4023,14 +4023,25 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate', *,
     return res
 
 
-def directionalmean(samples, *, axis=0, normalize=True):
+DirectionalStats = namedtuple('DirectionalStats',
+                              ('mean_direction', 'mean_resultant_length'))
+
+
+def directional_stats(samples, *, axis=0, normalize=True):
     """
-    Computes the directional mean of a sample of vectors.
+    Computes the directional mean (also called the mean direction vector) and
+    length of the mean resultant vector of a sample of vectors.
 
     The directional mean is a measure of "preferred direction" of vector data.
-    It is analogous to the sample mean, but it is for use when the magnitude of
+    It is analogous to the sample mean, but it is for use when the length of
     the data is irrelevant (e.g. unit vectors).
 
+    The length of the mean resultant vector can be used to compute measures
+    of "directional spread" such as the directional variance
+    vector data. The length of the mean vector can be used to calculate
+    directional variance using one of the several definitions outlined in
+    [1]_ and [2]_.
```

Comment:
We seem to be concerned that people will not understand "mean resultant", so I'd rather not describe "mean resultant length" in terms of an undefined "mean resultant vector". 

```suggestion
    The mean resultant length is a value between 0 and 1 used to quantify the 
    dispersion of directional data: the smaller the mean resultant length, the
    greater the dispersion. Several definitions of directional variance
    involving the mean resultant length are given in [1]_ and [2]_.
```

I'd also suggest changing "length of the mean resultant vector" to "mean resultant length" throughout.

Reply:
Done.